cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -125,7 +128,7 @@ static String workingDir() {
 
     Pipeline.Context DEFAULT_CONTEXT = Context.DIAGNOSTIC;
 
-    static ImmutableArguments.Builder defaultsBuilder(String profileString) {
```

Comment:
small nit, but i prefer not to on-demand static import this, otherwise can be tricky to see right away what type of builder it is.